libs/langchain-community/src/document_loaders/web/firecrawl.ts

```diff
@@ -56,6 +62,7 @@ export class FireCrawlLoader extends BaseDocumentLoader {
     super();
     const {
       apiKey = getEnvironmentVariable("FIRECRAWL_API_KEY"),
+      apiUrl = "https://api.firecrawl.dev",
```

Comment:
Let's just leave this unset by default? Then we can only need to pass it to the underlying SDK if it is overridden?